tests/test_spahm_b.py

```diff
@@ -8,10 +8,53 @@ def test_water():
     xyz_in = path+'/data/H2O.xyz'
     mols = utils.load_mols([xyz_in], [0], [0], 'minao')
     dms = utils.mols_guess(mols, [xyz_in], 'LB', spin=[0])
-    X = bond.bond(mols, dms, spin=[0])
+    X = bond.get_repr(mols, [xyz_in], 'LB', spin=[0], with_symbols=False, same_basis=False)
```

Comment:
I just changed this test, so that we test `get_repr()` at least once.

And just adapted the other ones to the array returned by `bond()`
_Do you think it's ok? or should we test them individually (`get_repr()` and `bond()`)?_

Reply:
i guess it's fine